src/main/java/org/radarbase/management/web/rest/LoginEndpoint.kt

```diff
@@ -0,0 +1,34 @@
+package org.radarbase.management.web.rest
+
+import org.radarbase.management.config.ManagementPortalProperties
+import org.slf4j.LoggerFactory
+import org.springframework.beans.factory.annotation.Autowired
+import org.springframework.web.bind.annotation.GetMapping
+import org.springframework.web.bind.annotation.RestController
+import org.springframework.web.servlet.view.RedirectView
+
+@RestController
+class LoginEndpoint
+    @Autowired
+    constructor(
+        @Autowired private val managementPortalProperties: ManagementPortalProperties,
+    ) {
+        @GetMapping("/oauth/login")
```

Comment:
should we name these `api/redirect/login` and `api/redirect/account`?

Reply:
Yes made the changes.